### PR TITLE
Replace CSVReader & JsonRPC libs

### PR DESCRIPTION
**Pull request by tenzap**
They had no known license

### CODE REVIEW

**Comment by tenzap** [comment]
any idea for the error in ci? Update composer.json to an older version of league/csv lib? Which one? How to do that?

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires league/csv ^9.8 -> satisfiable by league/csv[9.8.0].
    - league/csv 9.8.0 requires php ^7.4 || ^8.0 -> your php version (7.2.34) does not satisfy that requirement.
```

**Comment by tenzap** [comment]
Ok but this means users with 7.0 won't be able to use kalkun because of composer? Is it possible to tell composer to allow older versions depending on php version ?

**Comment by kingster** [comment]
https://github.com/thephpleague/csv says that `9.2.0 ` version would support PHP version `7.0.10`. 

The way to tell composer to use either of the version is 

```   "league/csv" : "^9.2.0 || ^ 9.6.2 || ... "``` this way composer would choose the appropriate version based on the PHP version.

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
         "codeigniter/framework": "^3.1",
         "paragonie/random_compat": "^9.99",
         "giggsey/libphonenumber-for-php": "^8.12",
+        "league/csv": "^8.2 || ^9.5 || ^9.6 || ^9.7 || ^9.8",
```
Comment (kingster) [inline composer.json:16]: Could be simply written as "^8.2 || ^9.5" 

https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Comment (tenzap) [reply composer.json:16]: I'm not sure to understand.
8.2 needs php >= 5.5
9.5 needs php >= 7.0.10
9.6 needs php >= 7.2.5
9.7 needs php >= 7.3
9.8 needs php >= 7.4

So should I just use `"^8.2 || ^9.5"` and user will get the latest version his php version supports? For example:
- If he has php7.2.5 he will get up 9.6.2 then?
- If he has php7.0.10 he will get up 9.5.0 then?

Comment (kingster) [reply composer.json:16]: Yes, atleast that's what the documentation says.

We can keep an eye on any reported issue just in case.

Comment (tenzap) [reply composer.json]: updates, thanks.

**Review by kingster — APPROVED** [review]

**Pull request merged**